src/aqua/installation-spell/src/aqua/log.aqua

```diff
@@ -1,14 +1,13 @@
-import Op, Debug, Peer from "@fluencelabs/aqua-lib/builtin.aqua"
+import Op, Debug, Peer, Srv from "@fluencelabs/aqua-lib/builtin.aqua"
 import Spell from "@fluencelabs/spell/spell_service.aqua"
 
-import SpellData from "./spell_data.aqua"
 import Console from "./console.aqua"
 
 
 -- func console(any: T):
 func log(arg: ⊤):
-    spell_id <- SpellData.spell_id()
     str <- Debug.stringify(arg)
+    spell_id <- Srv.resolve_alias("spell")
```

Comment:
If you want to use `"spell"` alias, you can just do `Spell "spell"`, right?

Reply:
I need spell_id for logs!

Reply:
Maybe logs should add spell_id automatically